### ISSUE TITLE
highlighted tags no longer displaying correctly

### ISSUE THREAD

**Issue by DavidPostill** [soscripted/sox#84]
**Installed Version: 2.0.1  |  Environment: Firefox/greasemonkey**

"Add highlighted tags to closed/on hold/duplicate/migrated questions on question lists" the tags are not displaying correctly:

![screenshot - 13_08_2016 17_43_05](https://cloud.githubusercontent.com/assets/7079224/17644398/6a99b930-617d-11e6-8b8a-d3481edb7b38.png)

They no longer look like nice (coloured) buttons :/

**Comment by fixer1234** [comment]
@DavidPostill, I assume the image is from the main page? (my view is slightly different, votes/answers/views are aligned vertically.)   My environment is also Firefox/Greasemonkey.  On my systems, the feature still works (colored buttons), as before.  BTW, the tags in your image also don't appear in tag boxes as they do on mine.  Are you running any other scripts that affect page rendering (potential conflict with SOX)?

**Comment by DavidPostill** [comment]
Image is from the main page. However I see the issue on other pages as well (where the votes are aligned vertically). The tag boxes I think is a side effect of this bug https://meta.stackexchange.com/questions/283314/network-wide-change-to-question-tags (at least on so and meta.so. I see normal tag boxes on other sites but still incorrect rendering of the question "status" tags.

I have a bunch of other scripts running. The obvious candidate is SOUP but disabling that doesn't fix the "status" tags.

**Comment by shu8** [comment]
@DavidPostill Seems like the CSS hasn't loaded properly. Can you try doing a hard refresh? (Ctrl+F5)

**Comment by DavidPostill** [comment]
Done. No change.

**Comment by shu8** [comment]
@DavidPostill hmm, can you go to the dev tools (f12), and check the console if there is any error loading any pages? You'll need to enable the 'Net' button on the top-left.

**Comment by DavidPostill** [comment]
There is only one error:

![screenshot - 14_08_2016 12_20_36](https://cloud.githubusercontent.com/assets/7079224/17648781/8f47479c-6219-11e6-848b-82c3b2b3dc94.png)

Does that help?

**Comment by shu8** [comment]
@DavidPostill hmm, no :( Thanks anyway. I'll see if I can find out the cause ASAP :)

**Comment by shu8** [comment]
@DavidPostill Can you access [this CSS link](https://rawgit.com/shu8/SE-Answers_scripts/master/dupeClosedMigratedCSS.css)? And does it show a page with this on it:?

```
.duplicate, .closed, .migrated, .onhold{
    color: #FFF;
    padding: 2px;
    border-radius: 4px;
    font-size: 12px;
}
.duplicate {background: #FA0;}
.closed {background: #F00;}
.migrated {background: #19F;}
.onhold {background: #808080;}
```

---

Also, please could you open the console (F12), and then paste `document.styleSheets` into the console (at the bottom, next to the `>>`), and copy/paste the output of that here, or take a screenshot? It should look something like this:

![this](https://cloud.githubusercontent.com/assets/8850830/17675507/f30fa8c4-6321-11e6-868d-6d9cc3df9106.png)

**Comment by DavidPostill** [comment]
Yes. That link works and I see the CSS.

I don't see quit the same as you in my developer console:

![screenshot - 15_08_2016 20_26_54](https://cloud.githubusercontent.com/assets/7079224/17676525/e57ab49c-6326-11e6-925d-bdaa2b1d2d19.png)

![screenshot - 15_08_2016 20_27_45](https://cloud.githubusercontent.com/assets/7079224/17676528/ead52e18-6326-11e6-8d03-69217e4572c2.png)

**Comment by shu8** [comment]
@DavidPostill sorry for the late reply, could you try pasting this in instead please?:

```
function checkIfIncluded(file) { //http://stackoverflow.com/a/10419583/3541881
    var links = document.getElementsByTagName("link");
    for(var i = 0; i < links.length; i++) {
        if (links[i].href.substr(-file.length) == file)
            return true;
    }

    var scripts = document.getElementsByTagName("script");
    for(var i = 0; i < scripts.length; i++) {
        if (scripts[i].src.substr(-file.length) == file)
            return true;
    }

    return false;
}
checkIfIncluded('dupeClosedMigratedCSS.css');
```

and tell me whether the output is 'false' or 'true'?

**Comment by DavidPostill** [comment]
Output is 'true'

**Comment by shu8** [comment]
@DavidPostill could you join me [in chat](https://gitter.im/soscripted/sox?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) for a few minutes if at all possible?

**Comment by shu8** [comment]
If not, that's fine; when you get some time, please could you right click on any 'onhold'/'duplicate' text, click 'inspect element', and see on the right whether you see something like this:

![this](https://cloud.githubusercontent.com/assets/8850830/17908631/2fecce0e-6979-11e6-8c3d-ebd7447b00da.png)

If not, what do you see?

**Comment by DavidPostill** [comment]
![screenshot - 24_08_2016 8_19_21](https://cloud.githubusercontent.com/assets/7079224/17921754/7cd826e6-69d3-11e6-8d2e-d4e9ed2dff99.png)

It looks different to yours ...

**Comment by shu8** [comment]
@DavidPostill That's really weird... Do you have any style-blocker or style-changer extension installed? Can you try running SOX with them disabled?

A final check you could do is to make sure you see this:

![this](https://cloud.githubusercontent.com/assets/8850830/17937928/bd7d648e-6a1b-11e6-9b75-43080db3ee0e.png)

If you do, then there must be some extension blocking it :confused:

**Comment by DavidPostill** [comment]
Hmm. I don't see it ... times passes ... got it. It is [Privacy Badger](https://www.eff.org/privacybadger) blocking it. No idea why.

**Comment by shu8** [comment]
@DavidPostill aha! Could you change the privacy badger slider things to unblock it? I don't use it myself so not entirely sure how it works.

**Comment by DavidPostill** [comment]
Yes. That works. I had Privacy Badger in set and forget mode. They must have changed something a couple of weeks ago ... :/
